modules/dxp/apps/osb/osb-spring-boot/osb-spring-boot-client-zendesk/src/main/java/com/liferay/osb/spring/boot/client/zendesk/service/ZendeskService.java

```diff
@@ -218,6 +218,88 @@ public SearchHits<ZendeskTicket> search(
 		return toSearchHits(jsonObject);
 	}
 
+	public void updateZendeskOrganization(
+			ZendeskOrganization zendeskOrganization)
```

Comment:
I think it would be clearer if business events is passed as a parameter here cause that's all that's changing